__tests__/middleware.test.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 require('@code-fellows/supergoose');
-const auth = require('../auth/middleware.js');
+const auth = require('./auth/middleware.js');
```

Comment:
This is causing an error for your tests, should be switched back to:

`const auth = require('../auth/middleware.js')`